extensions/web/08_text_encoding.js

```diff
@@ -373,14 +388,16 @@
       if (BOMEncoding === "UTF-8") start = 3;
       else start = 2;
     }
-    return new TextDecoder(encoding).decode(bytes.slice(start));
+    return new TextDecoder(encoding).decode(
```

Comment:
should use `TextEncoderPrototypeDecode`

Reply:
ditto in text_encoding: https://github.com/denoland/deno/blob/8bab7d42e2f5ec657b18f37621ae83b65ab0741c/extensions/web/08_text_encoding.js#L391

Reply:
`TextEncoderPrototypeDecode` does not exist. Primordials are only for JS builtins

Reply:
> `TextEncoderPrototypeDecode` does not exist. Primordials are only for JS builtins

It's prone to prototype manipulation, and that is the purpose of the existance of the primordials in general. Or are they more narrow in their scope/goal?

I see these as things that should be created, because - although I haven't checked the relevant spec for this algorithm - I doubt it specifies performing an ECMAScript `Get` operation on the `TextDecoder` instance.

Reply:
Primordials are limited to JS builtins not external APIs (analogous to its name). Node.js doesn't do `TextEncoderPrototypeDecode` either

Reply:
If that is so, then all of my comments here besides the ones about DataView and TypedArray can be discarded/ignored.

Although, we should Deno aim to only do as little as Node.js had?